bundler/helpers/lib/functions.rb

```diff
@@ -163,6 +163,24 @@ def self.set_bundler_flags_and_credentials(dir:, credentials:,
     end
   end
 
+  def self.relevant_credentials(credentials)
+    [
+      *git_source_credentials(credentials),
+      *private_registry_credentials(credentials)
+    ].select { |cred| cred["password"] || cred["token"] }
+  end
+
+  def self.private_registry_credentials(credentials)
+    credentials.
+      select { |cred| cred["type"] == "rubygems_server" }
+  end
+
+  def self.git_source_credentials(credentials)
+    credentials.
+      select { |cred| cred["password"] || cred["token"] }.
```

Comment:
Minor but looks like this is already done for both under `relevant_credentials` so could ✂️ 
```suggestion
```